src/App.js

```diff
@@ -1,10 +1,30 @@
+// eslint-disable-next-line
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
- [ ] Kindly make sure to not disable Linters. Linters help us to write clean code.